list2/ex43/index.js

```diff
@@ -0,0 +1,17 @@
+export function combiningObjects() {
+  const obj1 = {
+    name: "Pedro",
+    age: 19,
+    city: "São Caetano do Sul",
+  };
+
+  const obj2 = {
+    age: 25,
+    city: "Rio de Janeiro",
+    salary: 2000,
+  };
+
+  const combinedObjects = Object.assign({}, obj1, obj2);
```

Comment:
Since ES6, the spread operator (...) is a more concise and modern way to merge objects. It is often preferred over Object.assign() for clarity and readability.

```js
const combinedObjects = { ...obj1, ...obj2 }
```